Gemfile

```diff
@@ -69,4 +68,5 @@ gem 'azure_mgmt_stream_analytics',          path: 'management/azure_mgmt_stream_
 gem 'azure_mgmt_subscriptions',             path: 'management/azure_mgmt_subscriptions'
 gem 'azure_mgmt_traffic_manager',           path: 'management/azure_mgmt_traffic_manager'
 gem 'azure_mgmt_web',                       path: 'management/azure_mgmt_web'
+gem 'azure_graph',                          path: 'data/azure_graph'
```

Comment:
should we call this gem graph_rbac? I'm wondering because there was another mention of Graph API (referring to Cosmos DB when we talked about samples) that was different recently.